lib/widgets/plan_widget.dart

```diff
@@ -0,0 +1,159 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:fokus/utils/app_locales.dart';
+import 'package:fokus/utils/theme_config.dart';
+
+class PlanWidget extends StatelessWidget {
+  final String planName, details;
+  final bool isActive, isInProgress;
+  final int tasksCount;
+  final double progressPercentage;
+  final Function buttonAction;
+
+  // styling vars
+  final double playButtonSize = 40.0;
+  final double tasksIconSize = 26.0;
+  final double progressIndicatorHeight = 15.0;
+  final double futurePlansButtonWidth = 80.0;
+  final int titleMaxLines = 3;
+  final Color activeButtonColor = Colors.teal;
+  final Color buttonIconColor = Colors.white;
+  final Color disabledButtonColor = Color.fromARGB(255, 200, 200, 200);
+  final Color inactiveProgressBar = Color.fromARGB(255, 240, 240, 240);
+  final Color inProgressButtonColor = Colors.amber;
+  final Icon notInProgressIcon = new Icon(Icons.play_arrow);
+  final Icon inProgressIcon = new Icon(Icons.launch);
+  final TextStyle blueTextStyle =
+      new TextStyle(fontSize: 16, color: Colors.blue);
+
+  PlanWidget(
+      {@required this.planName,
+      @required this.isActive,
+      @required this.isInProgress,
```

Comment:
This seems kind of excessive together with progressPercentage param. As plan in progress can still have 0% completion we could take a nullable int as progress. That way:
- the default (not passed) would mean it is not in progress
- any passed number means it is